app/javascript/src/components/Common/Sidebar/index.jsx

```diff
@@ -55,7 +55,7 @@ const Sidebar = () => {
       }}
       profileInfo={{
         name: `${user.first_name} ${user.last_name}`,
-        imageUrl: "https://i.pravatar.cc/300",
+        // imageUrl: "https://i.pravatar.cc/300",
```

Comment:
What's the point of pushing a commented code to master? This should've been caught in your self-review.